lib/membrane_rtc_engine/endpoints/hls/config/sink_bin_config.ex

```diff
@@ -17,7 +17,6 @@ if Code.ensure_loaded?(Membrane.HTTPAdaptiveStream.Manifest) do
             manifest_module: Manifest.t(),
             storage: (Path.t() -> Storage.config_t()),
             target_window_duration: pos_integer | :infinity,
-            persist?: boolean,
```

Comment:
This parameter isn't correlated with the `persist?` parameter from the mixer config.
```
              persist?: [
                spec: boolean,
                default: false,
                description: """
                If true, stale segments are removed from the manifest only. Once
                playback finishes, they are put back into the manifest.
                """
              ],
```